test/simulator/abilities/dancer.js

```diff
@@ -54,7 +54,8 @@ describe('Dancer', function () {
 	});
 
 	it('should not copy a move that failed or was blocked by Protect', function () {
-		battle = common.createBattle({gameType: 'doubles', seed: [1, 2, 3, 4]});
+		// hardcoded to RNG seed
```

Comment:
Why are you changing the seed again?

Blah, in general this file needs way more comments.

Reply:
I can look into this one later, but this is a long sequence of events that seems to be dependent on the RNG.

Reply:
Yeah, this is why I updated the test API to make it easier to tell what the sequence of events actually is... :/

Reply:
In any case, there's no need to change the seed with a call to `resetRNG` after the players join.